api-specificatie/openapi.yaml

```diff
@@ -121,30 +184,24 @@ paths:
     get:
       security:
         - apiKeyBAG: []
-      operationId: zoekAdres
+      operationId: raadpleegAdresMetNumId
       tags:
         - Adres
       summary: Raadpleeg adres o.b.v. een sleutel
-      description: "Raadpleeg het adres op basis van de nummeraanduiding identificatie verkregen via de zoeken-operatie
+      description: "Raadpleeg het adres met de nummeraanduiding identificatie verkregen via de zoek-operatie
         Geleverd worden de huidige gegevens van een adres evt. met embedded woonplaats, openbare ruimte, nummeraanduiding. 
         Huidige gegevens zijn actuele gegevens die geen eind status hebben, zoals 'Naamgeving ingetrokken'. 
         Als de nummeraanduiding is ingetrokken, dan levert een bevraging geen adres op. 
         Relaties met adresseerbaarobject (ligplaats, standplaats of verblijfsobject) en als het adresseerbaare object een verblijfsobject betreft het/de gerelateerde pand(en). 
-        Parameter nummeraanduidingIdentificatie bevat de 16 cijferige identificatie van een lnummeraanduiding.</li>
-        Als expand=openbareruimte dan wordt het gerelateerde openbare ruime object als geneste resources geleverd.<br/>
-        Als expand=nummeraanduiding dan wordt het gerelateerde nummeraanduiding object als geneste resources geleverd.<br/>
-        Als expand=woonplaats dan wordt het gerelateerde woonplaats object als geneste resources geleverd.<br/>
-        Als expand=true dan worden de gerelateerde openbare ruime, nummeraanduiding en woonplaats objecten als geneste resources geleverd.<br/>
-        De contentCrs in de header moet zijn epsg:28992"
+        Parameter nummeraanduidingIdentificatie bevat de 16 cijferige identificatie van een lnummeraanduiding."
       parameters:
         - $ref: '#/components/parameters/nummeraanduidingIdentificatie'
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/api-version"
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/expand"
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
-        - $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/headers.yaml#/contentCrs'
       responses:
         '200':
-          description: Bevraging zoekAdres geslaagd
+          description: Bevraging raadpleegAdresMetNumId geslaagd
```

Comment:
graag veranderen in: raadpleegAdres geslaagd. Dat je een identificatie als padparameter gebruikt is logisch.